README.md

```diff
@@ -154,6 +154,21 @@ $object = $bucket->object('file_backup.txt');
 $object->downloadToFile('/data/file_backup.txt');
 ```
 
+#### Stream Wrapper
+
+```php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Storage\StorageClient;
+
+$storage = new StorageClient([
+    'projectId' => 'my_project'
```

Comment:
It will work without it by relying on the Application Default Credentials. However, this was the way it was documented in previous sections of the README. Perhaps, we should add a section specifically about configuring the clients of each type (separate PR).